apps/sms/views/conversation/test/unit/information_test.js

```diff
@@ -366,8 +367,14 @@ suite('Information view', function() {
           sizeContent
         );
       }
-
-      sinon.assert.called(reportView.renderContactList);
+      if (receivers) {
+        sinon.assert.calledWithMatch(
+          reportView.renderContactList,
+          receivers.concat('sender')
+        );
+      } else {
+        sinon.assert.calledWith(reportView.renderContactList);
+      }
```

Comment:
@julienw , I don't think that's going to be simple. As we are not passing any `receivers` in 7 of the 8 tests, I'm getting something like this
```
Error: expected renderContactList to be called with match ["receiver"]
    renderContactList([{
  infoBlock: <div class="network-status" data-delivery-status="delivered"><span class="icon"><[...]</div>,
  number: "receiver"
}])
```
The 8 tests are expecting really different parameter. So I don't think it is going to be easy to generalize.